modular_skyrat/modules/event_vote/code/event_vote.dm

```diff
@@ -487,7 +489,7 @@
 			var/time
 			if(alert == "Yes")
 				time = tgui_input_number(usr, "Input custom time in seconds", "Custom time", 60, 6000, 1) * 10
-			if(!check_rights(R_PERMISSIONS))
+			if(!check_rights(R_FUN))
 				return
```

Comment:
This should be moved right after the `if("reschedule_low_chaos")` line, to be in line with the other ones (and to not allow for more access to those that shouldn't have access to it).

Also just this portion as a whole feels like it should be slightly re-written to make more sense, that code looks _really_ weird to me.